install.sh

```diff
@@ -138,17 +138,17 @@ install_packages( )
 
   # dependencies needed to build the rocblas library
   local library_dependencies_ubuntu=( "make" "cmake-curses-gui" "pkg-config"
-                                      "python2.7" "python3" "python-yaml" "python3-yaml"
+                                      "python2.7" "python3" "python-yaml" "python3-yaml" "python3-distutils"
                                       "llvm-6.0-dev" "zlib1g-dev" "wget")
   local library_dependencies_centos=( "epel-release"
                                       "make" "cmake3" "rpm-build"
-                                      "python34" "PyYAML" "python3*-PyYAML"
+                                      "python34" "PyYAML" "python3*-PyYAML" "python3*-distutils"
                                       "gcc-c++" "llvm7.0-devel" "llvm7.0-static"
                                       "zlib-devel" "wget" )
   local library_dependencies_fedora=( "make" "cmake" "rpm-build"
-                                      "python34" "PyYAML" "python3*-PyYAML"
+                                      "python34" "PyYAML" "python3*-PyYAML" "python3*-distutils"
                                       "gcc-c++" "libcxx-devel" "zlib-devel" "wget" )
-  local library_dependencies_sles=(   "make" "cmake" "python3-PyYAM"
+  local library_dependencies_sles=(   "make" "cmake" "python3-PyYAM" 
```

Comment:
The trailing space should be automatically removed by `pre-commit` -- you must run `install.sh` at least once to install `pre-commit`.